src/tic-tac-toe.cpp

```diff
@@ -5,13 +5,47 @@
 
 using namespace std;
 
+/* class Field {
+public:
+    Field (int n, int m) {
+        
+    }
+
+    bool makeMove(Player* player, int x, int y) {
+        
+    }
+
+    int status(Player* player) { // 2 -- win, -2 -- fail, 0 -- the game in progress 1 -- draw, -1 -- other;
+        
+    }
+
+private:
+    vector<vector<Player*>> d_grid;
+}; */
+
+class Player {
+public:
+    Player(string p_name, bool p_isCross) {
+        name = p_name;
+        isCross = p_isCross;
+    }
```

Comment:
Use constructor initializer list
```suggestion
    Player(string name, bool isCross) 
    : d_name(name),
    d_isCross(isCross)
    {
    }
```